Ubuntu/History/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ set(plugin_SRCS
     historyqmltexteventattachment.cpp
```

Comment:
In terms of security, what do we need to think about here? Can any app with access to history-service delete any and all messages or phone conversations now?

Reply:
Yes was thinking of that too. How can we only allow core apps ?
Btw today we can delete all by javascript, by querying events and use the removeEvents method